### ISSUE TITLE
Deno panics when installing modules and running a brand new nest.js project

### ISSUE THREAD

**Issue by max-abclabs** [denoland/deno#16663]
To reproduce:
1. Run `nest new <project-name>`
2. cd into project and refactor file-extensions and imports
3. run `deno run src/main.ts` or `deno cache src/main.ts`
4. Deno panics

```
~/Work/deno-nest-test » deno cache --reload src/main.ts

Download https://registry.npmjs.org/@nestjs/common
Download https://registry.npmjs.org/@nestjs/core
Download https://registry.npmjs.org/cache-manager
Download https://registry.npmjs.org/class-transformer
Download https://registry.npmjs.org/class-validator
Download https://registry.npmjs.org/iterare
Download https://registry.npmjs.org/reflect-metadata
Download https://registry.npmjs.org/rxjs
Download https://registry.npmjs.org/tslib
Download https://registry.npmjs.org/uuid
Download https://registry.npmjs.org/@nestjs/microservices
Download https://registry.npmjs.org/@nestjs/platform-express
Download https://registry.npmjs.org/@nestjs/websockets
Download https://registry.npmjs.org/@nuxtjs/opencollective
Download https://registry.npmjs.org/fast-safe-stringify
Download https://registry.npmjs.org/object-hash
Download https://registry.npmjs.org/path-to-regexp

============================================================
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: macos aarch64
Version: 1.28.0
Args: ["deno", "cache", "--reload", "src/main.ts"]

thread 'main' panicked at 'assertion failed: !old_node_children.contains_key(next_specifier)', cli/npm/resolution/graph.rs:855:7
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by glebbash** [comment]
Minimal reproduction is the same as in #16629

**Comment by bartlomieju** [comment]
This has been already fixed in https://github.com/denoland/deno/pull/16641 and will be released in v1.28.1
